scripts/quests/ahtUrhgan/Ode_to_the_Serpents.lua

```diff
@@ -0,0 +1,76 @@
+-----------------------------------
+-- Ode to the Serpents
+-----------------------------------
+-- Log ID: 6, Quest ID: 44
+-- Fari-Wari: !pos 80 -6 -137 50
+-----------------------------------
+require("scripts/globals/items")
+require("scripts/globals/quests")
+require("scripts/globals/npc_util")
+require('scripts/globals/interaction/quest')
+require("scripts/globals/zone")
+-----------------------------------
+
+local quest = Quest:new(xi.quest.log_id.AHT_URHGAN, xi.quest.id.ahtUrhgan.ODE_TO_THE_SERPENTS)
+
+quest.reward =
+{
+    item = xi.items.IMPERIAL_GOLD_PIECE,
+}
+
+quest.sections =
+{
+
+    {
+        check = function(player, status, vars)
+            return status == QUEST_AVAILABLE and
+            player:getQuestStatus(xi.quest.log_id.AHT_URHGAN, xi.quest.id.ahtUrhgan.SAGA_OF_THE_SKYSERPENT) == QUEST_COMPLETED
```

Comment:
This can be simplified to `player:hasCompletedQuest(xi.quest.log_id.AHT_URHGAN, xi.quest.id.ahtUrhgan.SAGA_OF_THE_SKYSERPENT)`